doc/api/events.md

```diff
@@ -296,13 +296,17 @@ Its `name` property is set to `'MaxListenersExceededWarning'`.
 <!-- YAML
 added: v0.1.26
 -->
+- `eventName` {string|symbol}
```

Comment:
We're more lax than this. We basically accept any value, it's just that (currently) it will be converted to a string before being used. So for example numbers, booleans, functions (the function's source will be used as the event name), etc. all work.